config/routes.rb

```diff
@@ -1,3 +1,10 @@
 Rails.application.routes.draw do
   # For details on the DSL available within this file, see https://guides.rubyonrails.org/routing.html
+  root "about_us#home"
+  get 'about_us/about_us', to: 'about_us#about_us'
+  get 'about_us/contact', to: 'about_us#contact_us'
+  resources :events, only: [:index, :new, :show]
+  resource :sessions, only: [:new, :delete]
+  resources :users, only: [:new]
+  resources :subscribers, only: [:create]
```

Comment:
should there also be delete for subscribers who want to unsubscribe?

Reply:
No. I dont think so. We would have to authenticate the subscriber which adds an entirely new level of complication.